data/json/overmap/overmap_terrain/overmap_terrain_hardcoded.json

```diff
@@ -257,6 +257,16 @@
     "see_cost": 5,
     "flags": [ "NO_ROTATE" ]
   },
+  {
+    "type": "overmap_terrain",
+    "id": "deep_rock",
+    "//": "Ugly placeholder solution to differentiate between subway tunnels, remove if per-special connection subtyping is implemented.",
+    "name": "solid rock",
```

Comment:
Check #49823 - "looks_like": "empty_rock" would be appropriate, I suspect (you might need to update to pull in that commit to test locally, of course!).

Reply:
Thanks for pointing that out, keeping up with the post-stable flurry is a challenge. I'll need a bit to come up with some ambience snippets since those are a thing now, so I'll wait with the rebase fun until I'm properly done I think.

Reply:
I apparently missed the "ambience snippets" - or are you meaning things like #48244 that Ramza13's eoc work allows for?